Public/Sdk/Public/Transformers/Transformer.ReadPipGraphFragment.dsc

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Transformer {
+
+    /**
+     * Read in a fragment of the graph as a binary file and add all pips.
+     * name: Name of this pip graph fragment, used to return a handle to it.
+     * file: Binary file to read.
+     * dependencyFragments: All other pip graph fragments this one depends on.
+		 * description: String to print to console while fragments are loading.

```

Comment:
>description [](start = 5, length = 11)

This one should be optional. All other pip constructs have this as an optional field where we print a useful default.